integration-tests/README.md

```diff
@@ -0,0 +1,25 @@
+# CLP Integration Tests
+
+This Python module provides end-to-end tests for the CLP package using the `pytest` framework.
+It covers package lifecycle scripts and validates compression and search functionality for
+performance and correctness on large [datasets].
+
+
+## Running tests
+To build the package and run all integration tests:
+```
+task package-test
```

Comment:
```suggestion
task tests:integration
```

Reply:
We could have a more specific task to test just core (`tests:integration:core`).